smithy-protocol-tests/build.gradle

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+plugins {
+    id "software.amazon.smithy.gradle.smithy-jar" version "$smithyGradleVersion"
+}
+
+description = "Defines protocol tests for Smithy HTTP protocols."
+
+ext {
+    displayName = "Smithy :: Protocol Tests"
```

Comment:
Should this be updated to be specific to cbor? Same with the modulename.

Reply:
These aren't specific to RPCv2 CBOR.